src/components/NotePage/NoteList/NoteItem.tsx

```diff
@@ -70,27 +70,68 @@ export const StyledNoteListItem = styled.div`
     border-radius: 13px;
     display: inline-block;
   }
+
+  .highlighted {
+    background: rgba(3, 197, 136, 0.6);
+  }
 `
 
 type NoteItemProps = {
   note: NoteDoc
   active: boolean
   storageId: string
+  search: string
   basePathname: string
   focusList: () => void
 }
 
-export default ({ storageId, note, active, basePathname }: NoteItemProps) => {
+export default ({
+  storageId,
+  note,
+  active,
+  basePathname,
+  search
+}: NoteItemProps) => {
   const href = `${basePathname}/${note._id}`
 
+  const getHighlightedText = useCallback(
```

Comment:
Accept 2 arguments(search and text) and extract from this FC(functional component).